src/org.freedesktop.fwupd.xml

```diff
@@ -307,6 +307,17 @@
       </arg>
     </method>
 
+    <!--***********************************************************-->
+    <method name='EnableVerbose'>
```

Comment:
Is the logic we can enable it, but never disable it?

Reply:
That was my thought, but if you think it should be disabled when the client disconnects I can change that.

Reply:
I think we need to do two things: make it disable when the client disconnects, and probably use a different flag for daemon verbose, rather than just hijacking `--verbose` in the client.

Reply:
OK.